ipfshttpclient/client/pin.py

```diff
@@ -67,11 +69,14 @@ def ls(self, type="all", **kwargs):
 		-------
 			dict : Hashes of pinned IPFS objects and why they are pinned
 		"""
-		kwargs.setdefault("opts", {})["type"] = type
+		opts = {
```

Comment:
Add this comment above: `#PY2: No support for kw-only parameters after glob parameters`

Reply:
Added.